src/client/utils/Intercom.tsx

```diff
@@ -23,7 +23,8 @@ const createIntercom = () => {
 export const Intercom: React.FC = () => {
   const variation = useVariation()
   const isIntercomMatch = useRouteMatch(
-    LOCALE_PATH_PATTERN + '/new-member/:place(offer|sign|download)',
+    LOCALE_PATH_PATTERN +
+      '/new-member/:place(offer|sign|download|connect-payment)',
   )
   const isDirectConnectPaymentIntercomMatch = useRouteMatch(
     LOCALE_PATH_PATTERN + '/new-member/connect-payment/direct',
```

Comment:
Is this thing needed anymore? Seems when I tested it locally like it's not, since this case is covered by the addition `connect-payment` to the `LOCALE_PATH_PATTERN`?

Reply:
You are correct! 💯